crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/XmlProcessor.scala

```diff
@@ -3,14 +3,14 @@ package com.gu.crossword.crosswords
 import java.util.Locale
 import scala.xml._
 import org.joda.time.format.{ DateTimeFormat, ISODateTimeFormat }
-import org.joda.time.{ DateTimeZone, LocalDateTime }
+import org.joda.time.{ DateTimeZone, LocalDateTime, DateTime }
 
 trait DateLogic {
 
   def transformDate(dateString: String): String = {
-    val inputFormat = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm")
-    val outputFormat = ISODateTimeFormat.dateTime()
-    val isoFormattedOutput = LocalDateTime.parse(dateString, inputFormat).toDateTime(DateTimeZone.forID("Europe/London")).toString(outputFormat)
+    val inputFormat = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm").withZone(DateTimeZone.forID("Europe/London"))
+    val outputFormat = ISODateTimeFormat.dateTime().withZone(DateTimeZone.forID("UTC"))
+    val isoFormattedOutput = DateTime.parse(dateString, inputFormat).toString(outputFormat)
     isoFormattedOutput.replaceAllLiterally("Z", "+00:00")
```

Comment:
I realise it's not part of this change, but doing a string replace on the output is a bit odd. Better to just use the right format in the first place.

This should work:
```scala
DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZZ")
```

see [Zone in this docs page](http://joda-time.sourceforge.net/apidocs/org/joda/time/format/DateTimeFormat.html)